packages/bako-safe/src/constants.ts

```diff
@@ -0,0 +1,10 @@
+export const APP_NAME = 'Bako Safe';
+export const APP_DESCRIPTION =
+  'Bako Safe is a connector to safe.bako.global, a non-custodial vault for your crypto assets.';
+
+export const APP_IMAGE_DARK =
+  'https://besafe-asset.s3.amazonaws.com/BAKO_CONNECTOR_ICON.svg';
+export const APP_IMAGE_LIGHT =
+  'https://besafe-asset.s3.amazonaws.com/BAKO_CONNECTOR_ICON.svg';
+export const APP_BSAFE_URL = 'https://safe.bako.global';
+export const API_URL = 'https://api-multsig.infinitybase.com';
```

Comment:
For security measures all url's should be under a safe URL with HTTPS certificate on the name of the project. This means all URLs should be `<name>.bako.global`.

<img width="440" alt="Screenshot 2024-03-20 at 15 36 31" src="https://github.com/FuelLabs/fuel-connectors/assets/3941923/62c919b0-2ad6-4c89-8c6d-c5c5565db822">